public/regdesk.html

```diff
@@ -42,6 +42,8 @@
                 <span id="adultPrintIcon">❌</span> Adult <span id="adultPrintName">(No Printer)</span>
               </a>
               <ul class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                <li id="adultPrintManual"><a class="dropdown-item" href="#" data-bs-toggle="modal" data-bs-target="#manualPrintModal">Manual Label</a></li>
```

Comment:
Should this be an action to match all the other items in the list? `Print Manual Label`

Reply:
Good point, slightly modified to "Make" as you don't directly print something, you open a form to make the label and then click print.